app/va/va_profile/va_profile_client.py

```diff
@@ -157,8 +158,16 @@ def get_mobile_telephone_from_contact_info(self, contact_info: ContactInformatio
             )
 
         if telephone.get('countryCode') and telephone.get('areaCode') and telephone.get('phoneNumber'):
-            self.statsd_client.incr('clients.va-profile.get-telephone.success')
+            self.statsd_client.incr('clients.va-profile.get-telephone.failure')
```

Comment:
Why was this changed to "Failure?" It looks like a success to me

Reply:
Copy paste error, nice catch! [Updated](https://github.com/department-of-veterans-affairs/notification-api/pull/2034/commits/b7ae73e19b44d51402d3fab9f4eac620642ceaf3).